app-backend/db/src/test/scala/com/azavea/rf/database/SceneWithRelatedDaoSpec.scala

```diff
@@ -65,12 +65,12 @@ class SceneWithRelatedDaoSpec extends FunSuite with Matchers with Checkers with
             datasource1 <- DatasourceDao.create(dsCreate1.toDatasource(dbUser1), dbUser1)
             datasource2 <- DatasourceDao.create(dsCreate2.toDatasource(dbUser2), dbUser2)
             dbScenes1 <- (scenes1 map {
-              (scene: Scene.Create) => fixupSceneCreate(dbUser1, datasource1, scene)
+              (scene: Scene.Create) => fixupSceneCreate(dbUser1, datasource1, scene).copy(visibility = Visibility.Private)
```

Comment:
Sorry for the drive by, but if we want these always to be `PRIVATE`, can we change the generator
instead of copying after the fact? That'd be a `Gen.const(Visibility.Private)` [here](https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/datamodel/src/test/scala/com/azavea/rf/datamodel/implicits/Generators.scala#L63-L64) or [here](https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/datamodel/src/test/scala/com/azavea/rf/datamodel/implicits/Generators.scala#L322), depending on whether we want `PRIVATE` for scenes only.

Reply:
Ah, I originally modified the visibility generator, but then thought better of it because I didn't want to force all the tests to be restricted this way. 

I will add the constant though for the scene generator, that seems like the best solution.